prometheus_api_client/prometheus_connect.py

```diff
@@ -40,7 +40,16 @@ def __init__(
             self, url: str = "http://127.0.0.1:9090", headers: dict = None,
             disable_ssl: bool = False, retry: Retry = None
     ):
-        """Constructor for the class PrometheusConnect."""
+        """
+        Constructor for the class PrometheusConnect.
+
+        :param url: (str) Prometheus URL
+        :param headers: (disc) Custom headers
+        :param disable_ssl: (bool) Use insecure connection
+        :param retry: (Retry) Retry adapter to retry on HTTP errors
```

Comment:
Could you please add this to the class doc ([here](https://github.com/AICoE/prometheus-api-client-python/blob/master/prometheus_api_client/prometheus_connect.py#L26-L34)), instead of the Constructor doc?

Reply:
sure